### ISSUE TITLE
unable to serve CSS - same as #51, path confusion

### ISSUE THREAD

**Issue by mooseyboots** [Shirakumo/radiance#55]
hi, 

i'm trying out radiance, as per the readme getting started instructions, and it's unclear how the local `/static/` CSS should work.

if i spin up a new module from scratch, it only renders CSS if i make the path for the `uri-to-url` function is given as `/static/example/example.css`, but the relative file path actually on disk is `/static/example.css`.

i.e. it doesn't work if both paths are the same, i.e both `/static/example.css` or both `/static/example/example.css`, it only works if they don't match.

which is very confusing, to this total noob at least. is this deliberate somehow? shouldn't all css files under `/static/` just be read so the subdir doesn't matter in this way? that's at least what i'm used to from elsewhere. if we are already in our module directory, do we need a subdir below static to specify that we are also in our module directory?

while i'm here, is this always a strictly relative path? asking as i want to change `*modules-directory*`.

thanks.

**Comment by Shinmera** [comment]
It is deliberate, because you share the `/static` path as it's served with all other modules, not just your own.

**Comment by Shinmera** [comment]
`*modules-directory*` does not influence anything at all except where the scaffold generator stores stuff by default.

**Comment by mooseyboots** [comment]
now i'm trying under a different modules dir, path `/static/rad-test/rad-test.css`, file location `/static/rad-test.css` and it also doesn't render...

**Comment by Shinmera** [comment]
is your module actually called `rad-test`?

**Comment by mooseyboots** [comment]
the one under a different path is yes

**Comment by Shinmera** [comment]
And is it known to radiance? Is it actually loaded?

**Comment by mooseyboots** [comment]
yes, i would figure so, because the page text loads, & updates if i change it.

**Comment by Shinmera** [comment]
That's not a given. Did you quickload or load-system your module?

**Comment by mooseyboots** [comment]
`(ql:quickload "rad-test")` in the repl

**Comment by Shinmera** [comment]
You can check what path it actually looks under with `(radiance:static-file "rad-test.css" :rad-test)`

Anyway, the quick tutorial in the readme is extremely terse on purpose and does not serve as a proper introduction to everything. I'm not interested in making it into one, that's what the lengthy tutorial linked from the documentation is for.
